CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ _None_
   [@kevinstier](https://github.com/kevinstier)
   [#562](https://github.com/SwiftGen/SwiftGen/issues/562)
   [#768](https://github.com/SwiftGen/SwiftGen/pull/768)
+* IB: Added support for instantiating scenes while providing a `creator` block, commonly used for dependency injection. This feature is available starting from iOS 13, tvOS 13 and macOS 10.15.    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/9998c51351f6384bd7524010e61558cb9cf44506/CHANGELOG.md#L23"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>